backend/src/models/events.py

```diff
@@ -15,10 +17,9 @@ class Event(BaseModel, TimestampAbstractModel):
     date: Mapped[datetime] = mapped_column(DateTime(timezone=True), default=datetime.now, nullable=False)
     # TODO: should be implemented Enum choice field
     place: Mapped[str] = mapped_column(default="scena")
-    type: Mapped[list["Performance"]] = relationship(
-        secondary="user_theatrical_role_relationship",
-        back_populates="actor",
-        lazy="joined",
-    )
-    # TODO: should be implemented Enum choice field
-    status: Mapped[str] = mapped_column()
+    is_approved: Mapped[bool] = mapped_column(unique=False, default=False)
+    performance_id: Mapped[int] = mapped_column(ForeignKey("performances.id"))
+    type: Mapped["Performance"] = relationship("Performance",
+                                                back_populates="events",
+                                                foreign_keys=[performance_id])
```

Comment:
```suggestion
    is_approved: Mapped[bool] = mapped_column(default=False)
    performance_id: Mapped[int] = mapped_column(ForeignKey("performances.id"), index=True)
    type: Mapped["Performance"] = relationship("Performance",
                                                back_populates="events",
                                                foreign_keys=[performance_id])
```
Probably it's worth adding b-tree index here.
And it's redundant to specify `unique=False` as it's `False` by default